src/app/modules/group/components/user-list/user-list.component.html

```diff
@@ -0,0 +1,48 @@
+<p-progressSpinner *ngIf="state === 'loading'"></p-progressSpinner>
+
+<p *ngIf="state === 'error'">Error while loading the group members</p>
+
+<p *ngIf="state === 'empty'">This group is empty, check below, the different ways to add users or sub-groups.</p>
+
+<alg-grid
+*ngIf="state === 'ready'"
+[data]="members"
+[showGear]="false"
+>
+  <ng-template #headerTemplate let-columns>
+    <tr>
+      <th>
+        <div class="header-container">
+          Name
+        </div>
+      </th>
+      <th>
+        <div class="header-container">
+          Login
+        </div>
+      </th>
+      <th>
+        <div class="header-container">
+          Member Since
+        </div>
+      </th>
+    </tr>
+  </ng-template>
+
+  <ng-template
+    #bodyTemplate
+    let-member
+  >
+    <tr class="p-selectable-row">
+      <td>
+        {{ member.user.firstName }} {{member.user.lastName}}
+      </td>
+      <td>
+        {{ member.user.login }}
+      </td>
```

Comment:
It is correct that it is not beautiful when first/last name is null/empty. Maybe you could merge is a single "name" column and, as we did at other places, do "Gabriel Vidal (gvi)" or "gvi" if no last/first name. Maybe you could do your own pipe not have to redefine once more this rule (and replace other places where is it already done).

Reply:
fixed, I created the `userDisplay` pipe for this. I still need to replace other place where this can be used.

Reply:
do you plan to do it in this PR? It is maybe better.

Reply:
I created a separate PR (#197) for the user pipe, as some refactoring had to be done.